doc/Tex/RAM_SCB_install.tex

```diff
@@ -187,41 +79,27 @@ \section{Installation, Configuration, and Compiling \label{subchap:install}}
 \end{verbatim}
 
 
-Although RAM-SCB will try to use reasonable defaults based on your system, 
-there are a number of flags that allow you to customize your installation:
+Although RAM-SCB will try to use reasonable defaults based on your system, there are a number of flags that allow you to customize your installation:
 \begin{itemize}
-\item{Use {\tt -compiler} to select the Fortran 90 compiler.  
-  Common choices include pgf90, ifort, gfortran, and f95(for NAG).}
-\item{The {\tt -mpi} flag allows the user to pick which version of MPI to use.
-  Choose from mpich, mpich2, Altix, and openmpi.  Alternatively, the 
-  {\tt -nompi} flag may be set with no value.  This option is fragile.}
-\item{Set {\tt -ncdf} to the path of the NetCDF library installation.  
-  If used, this flag overrides the environment variable NETCDFDIR.}
-\item{Set {\tt -pspline} to the path of the Pspline library installation.  
-  If used, this flag overrides the environment variable PSPLINEDIR.}
+\item{Use {\tt -compiler} to select the Fortran 90 compiler. Common choices include pgf90, ifort, gfortran, and f95(for NAG).}
+\item{The {\tt -mpi} flag allows the user to pick which version of MPI to use. Choose from mpich, mpich2, Altix, and openmpi. Alternatively, the {\tt -nompi} flag may be set with no value. This option is fragile.}
```

Comment:
Same as previous comment here. I'd list in order that we're interested in using or supporting. Probably `openmpi`, then `mpich/mpich2`, then `altix`

Reply:
Done